jorvik/storage/isolation.py

```diff
@@ -0,0 +1,329 @@
+from typing import Callable
+import datetime
+
+from pyspark.sql import DataFrame, SparkSession, functions as F
+from pyspark.sql.streaming import StreamingQuery
+
+from delta.tables import DeltaTable
+
+from . import Storage
```

Comment:
No need to inherit from Storage and still you cant import from `__init__` change to relative import does not fix the issue.